data/governments.txt

```diff
@@ -583,6 +639,9 @@ government "Quarg"
 		"Hai" .01
 		"Hai (Wormhole Access)" .01
 		"Pirate" -.01
+		"Gegno" 1
+		"Gegno Vi" 1
+		"Gegno Scin" 1
```

Comment:
I see that no ship events can currently raise reputation with these factions, but should these still not be `.01`? Unless you want damaging your reputation with them to permanently damage your reputation with the Quarg? I suppose they wouldn't take too kindly to you attacking a lower tech group.

Reply:
While that sounds like something that makes sense, this would be too fast. I just got to government changes today (and am still learning about them), so I'm still fiddling with them, and this response helps with that a lot. Thank you.